loading-button-android/src/main/res/values/attrs.xml

```diff
@@ -6,5 +6,6 @@
         <attr name="spinning_bar_padding" format="dimension"/>
         <attr name="initialCornerAngle" format="dimension"/>
         <attr name="finalCornerAngle" format="dimension"/>
+        <attr name="use_drawable_cache" format="boolean"/>
```

Comment:
No need for this, this library should never use cache for the done animation. The cache was just a mistake that passed me by when refactoring the new version of the library. Let's just use the "no cache" as the only option.